doc/cephfs/administration.rst

```diff
@@ -3,6 +3,24 @@
 CephFS Administrative commands
 ==============================
 
+.. toctree:: 
+   :maxdepth: 1
+   :hidden:
+
+    Create a CephFS file system <createfs>
+	Provision/Add/Remove MDS(s) <add-remove-mds>
+    MDS failover and standby configuration <standby>
+    MDS Cache Size Limits <cache-size-limits>
+    MDS Configuration Settings <mds-config-ref>
+    Manual: ceph-mds <../../man/8/ceph-mds>
+    Export over NFS <nfs>
+    Export over NFS with volume nfs interface <fs-nfs-exports>
+    Application best practices <app-best-practices>
+    FS volume and subvolumes <fs-volumes>
+    CephFS Quotas <quota>
+    Health messages <health-messages>
+    Upgrading old file systems <upgrading>
```

Comment:
I'm looking at https://ceph--37616.org.readthedocs.build/en/37616/cephfs/administration/ and don't see all the entries above. It also seems to mix the sub-sections on this Administration page with the entries above which is kinda confusing.

Reply:
You're right.

I've spent the weekend learning how Sphinx deals with toctrees and sections, and as far as I can tell, toctrees and sections don't mix. So we'll have to figure out whether we want this document to be toctrees or sections (or we can find some way of integrating the raw-HTML hack with top-level rst files that top-level sidebar menu links can successfully target). @batrick , do you have an hour this week to discuss this? I don't want to just start hacking on your filesystem guide without consulting you and making sure that we're on the same page.

Reply:
I personally don't see the value in the sections on the side-bar TOC. Linking to sections is of course useful but no need for it to show in the side-bar. (We already have the convenient links to click `¶`when hovering over a section title.)

Reply:
As for a meeting, we can meet if you like. How about you come by the CephFS watercooler tomorrow at 6:45am PDT (check ceph community calendar). We can discuss your plans with the team.

cc @ajarr (we don't have anything scheduled already, right?)

Reply:
> As for a meeting, we can meet if you like. How about you come by the CephFS watercooler tomorrow at 6:45am PDT (check ceph community calendar). We can discuss your plans with the team.
> 
> cc @ajarr (we don't have anything scheduled already, right?)

@zdover23 @batrick  we can discuss this during our watercooler today at 6:45 AM PDT. There is nothing else scheduled.

Reply:
I only just saw this, and since Patrick isn't crazy about the sections in the side-bar TOC, I'm going to chew on this a little longer.  I've missed the meeting by ten minutes (but that's fine with me because I wasn't planning on being up this late and I'm completely useless right now anyway, being so tired).

The original point of this PR was to remedy the broken anchor links on the top-level side-bar TOC entries. Jos Collin had brought this to my attention, and I immediately saw the merit in his complaint. The more I engage with the CephFS documentation, though, the more I see Patrick's point, and the more I think that it's not as broken as I at first thought.